packages/remix-react/components.tsx

```diff
@@ -679,9 +680,11 @@ export function Meta() {
             )
           )
         ) : isOpenGraphTag ? (
-          <meta key={name} property={name} content={value} />
+          <meta property={name} content={value as string} />
+        ) : typeof value === "string" ? (
+          <meta name={name} content={value} />
         ) : (
-          <meta key={name} name={name} content={value} />
+          <meta {...value} />
```

Comment:
We still need keys on all elements, as we're in a loop.

Just a question on code readability: wouldn't it be better to break that giant ternary operation into something like:

```js
if (["charset", "charSet"].includes(name)) {
  return <meta key="charset" charSet={value} />
}

if (name === "title") { 
 // ...
```

Reply:
I think if I were going to write this code from scratch I would do the if/else myself.

Reply:
> We still need keys on all elements, as we're in a loop.

@machour This part isn't in the loop, so we can omit the `key` prop

> I think if I were going to write this code from scratch I would do the if/else myself.

@kentcdodds I'll refactor it to use the separate `if`-statements